Doppler.ContactPolicies.Business.Logic/Services/ContactPoliciesService.cs

```diff
@@ -0,0 +1,25 @@
+using System.Threading.Tasks;
+using Doppler.ContactPolicies.Business.Logic.DTO;
+using Doppler.ContactPolicies.Business.Logic.Extensions;
+using Doppler.ContactPolicies.Data.Access.Entities;
+using Doppler.ContactPolicies.Data.Access.Repositories.ContactPoliciesSettings;
+
+namespace Doppler.ContactPolicies.Business.Logic.Services
+{
+    public class ContactPoliciesService : IContactPoliciesService
+    {
+        private readonly IContactPoliciesSettingsRepository _settingsRepository;
```

Comment:
What do you think about a more explicit name? 
```suggestion
        private readonly IContactPoliciesSettingsRepository _contactPoliciesSettingsRepository;
```